tailwind.config.js

```diff
@@ -10,21 +10,8 @@ module.exports = {
   ],
   theme: {
     fontFamily: {
-      sans: [
```

Comment:
how about we only keeps these two fonts for hero section title, it looks not balanced between Chinese and English now